gatsby/data.rst

```diff
@@ -121,47 +126,66 @@ Change our index.js page like this:
         }
     `
 
-As you can see, first of all, we imported a new module `graphql`. This is used on the bottom of the file,
-to generate our query.
+First of all, we imported a new module `graphql`. This is used on the bottom of the file, to generate the query.
+
+When we add a GraphQL query in our page component, the result is passed to the component as property called `data`.
+
+In that property, we have the result of the query (with the same data structure).
 
-When we add a GraphQL query in our component, the result is automatically passed to the component as a prop called `data`.
+.. note::
+    To see what informations are in `data` property, try to put a ``console.log(data)`` in the component.
 
-In that prop, we have the required informations, in the same data structure of the query.
+    To do that, we need to change the returned value of the arrow function, because () automatically returns everything is inside them, and we wants to add some logic before returning the value. So the change should be like this:
 
-This method could be used in every page component, but if we break up our layout in several pieces (components) like we do in the previous chapter with the header,
-we need to use a different approach using `StaticQuery` components.
+    .. code-block:: none
```

Comment:
i tried different approaches like js, and javascript but i have this error when i try to compile it:

    WARNING: Could not lex literal_block as "javascript". Highlighting skipped.

But it should recognize it..am i missing something?

Reply:
@cekk according to jslint (online tool) this is not valid, here the message:

``Expected '/'.
        <h4>This is the site title: {data.site.siteMetadata.title}</h3>
``

One reason could be that you are using ``...``

One thing to remember,linters (for code blocks) expect usually valid code snippets :)

Since I do not speak `js` I can only tell you what the linter is telling me :), I have no idea if there is a difference for gatsby :)

Reply:
i was thinking that the error in console was more something like "i don't know how to interpret a javascript code-block" than "there is an error in your javascript block".

i'll try to fix it

Reply:
@cekk see [previous comment](https://github.com/plone/training/pull/341#discussion_r228316694)